libraries/AP_DDS/wscript

```diff
@@ -36,6 +36,8 @@ def configure(cfg):
     cfg.env.append_value('GIT_SUBMODULES', 'Micro-XRCE-DDS-Client')
     cfg.env.append_value('GIT_SUBMODULES', 'Micro-CDR')
 
+    cfg.env.AP_LIB_EXTRA_CXXFLAGS = ['-DSOME_CXX_FLAG']
+    cfg.env.AP_LIB_EXTRA_CFLAGS = ['-DSOME_C_FLAG']
```

Comment:
Since it's built as C++, I think this line can be removed.

Reply:
the idea was to add a general mechanism, not just for DDS